com.unity.netcode.gameobjects/Runtime/NetworkVariable/Collections/NetworkList.cs

```diff
@@ -63,6 +63,11 @@ public override bool IsDirty()
             return base.IsDirty() || m_DirtyEvents.Length > 0;
         }
 
+        internal void ConsiderDirty()
```

Comment:
What's tricky is that NetworkList, for historical reasons, uses the `m_IsDirty` flag to mean "needs full retransmit". Having `m_DirtyEvents`, on the flipside, means that we have content to send. I'm not willing to refactor it all, so I need a name that conveys "Well, it's kinda considered dirty to the base class, but we're not willing to set the full `m_IsDirty` flag". Maybe `MarkDirtyForBase` ?

Reply:
`.MarkNetworkObjectDirty()` I guess. I allows future devs to easily follow the path the call will make

Reply:
Yeah, I think MarkNetworkObjectDirty is good since it's explicit about what it's doing. I don't think a refactor is in scope for sure, but making the naming clearer helps with debugging as you mentioned.